src/05_lists.py

```diff
@@ -7,23 +7,32 @@
 # For the following, DO NOT USE AN ASSIGNMENT (=).
 
 # Change x so that it is [1, 2, 3, 4]
-# YOUR CODE HERE
+x.append(4)
 print(x)
 
 # Using y, change x so that it is [1, 2, 3, 4, 8, 9, 10]
-# YOUR CODE HERE
+x.extend(y)
 print(x)
 
 # Change x so that it is [1, 2, 3, 4, 9, 10]
-# YOUR CODE HERE
+x.remove(8)
 print(x)
 
 # Change x so that it is [1, 2, 3, 4, 9, 99, 10]
-# YOUR CODE HERE
+x.insert(5, 99)
 print(x)
 
 # Print the length of list x
-# YOUR CODE HERE
+print(len(x))
 
 # Print all the values in x multiplied by 1000
-# YOUR CODE HERE
\ No newline at end of file
+
+
+def times1000(num):
+    return num * 1000
+
+
+mapping = map(times1000, x)
+
+for num in mapping:
+    print(num)
```

Comment:
i like how you broke down the individual elements of this and separated them out. very good for readability. 

my solution was 
``` for 1 in x: 
        print(i * 1000) ```